src/model.rs

```diff
@@ -1313,6 +1316,7 @@ impl Event {
 				user_settings: try!(opt(&mut value, "user_settings", UserSettings::decode)),
 				user_server_settings: try!(opt(&mut value, "user_guild_settings", |v| decode_array(v, UserServerSettings::decode))),
 				tutorial: try!(opt(&mut value, "tutorial", Tutorial::decode)),
+				notes: try!(opt(&mut value, "notes", |v| decode_notes(v))),
```

Comment:
Writing `|v| decode_notes(v)` is redundant; you should be able to write `decode_notes` and get the same effect.